README.md

```diff
@@ -461,6 +461,11 @@ API context, this would include `PUT`, `POST`, `DELETE` and `PATCH`. This also
 includes the ability to perform SPARQL queries that perform updates, if those
 are supported.
 
+Write permissions on a container means the right to delete that container.
```

Comment:
This diverges from an earlier decision where `DELETE` requires write permissions on the container above it.

Reply:
Why is this about containers rather than resources?

Reply:
The story you're referring to is a good example of a case where the spec process failed, it's described in 'NB2' below.

Right, I only described the container case because that's where I was doubting myself, but it's a good idea to also describe non-container resources. Will update.

Reply:
> The story you're referring to is a good example of a case where the spec process failed

That is precisely why we should not spec it until there is agreement. The NBs show that this is too early to spec. Let's have a resolution before we write anything.

Reply:
OK, I'll mark this one as "still ambiguous what the current situation is", let's discuss it in the weekly meeting now!